DOC: add two tips for psql/pg_ctl DB management (#6344)
aiidateam/aiida-core: docs/source/intro/install_conda.rst

            (aiida) $ pg_ctl -D mylocal_db -l logfile start

        .. tip::

>             You may encounter issue that ``5432`` port is already in use. In this case, you can pass the ``-o "-F -p <port-num>"`` option to the ``pg_ctl`` command, ``<port-num>`` being the desired port number.
>             Then for the ``psql`` command, you can pass the ``-p <port-num>`` option.

Reviewer: ```suggestion
            The default port ``5432`` may already be in use by another process.
            In this case, you can pass the ``-o "-F -p <port-num>"`` option to the ``pg_ctl`` command, ``<port-num>`` being the desired port number.
            Then for the ``psql`` command, you can pass the ``-p <port-num>`` option.
```
